packages/tldraw/src/index.ts

```diff
@@ -391,6 +393,7 @@ export { useCanRedo, useCanUndo } from './lib/ui/hooks/menu-hooks'
 export { useMenuClipboardEvents, useNativeClipboardEvents } from './lib/ui/hooks/useClipboardEvents'
 export { useCopyAs } from './lib/ui/hooks/useCopyAs'
 export { useExportAs } from './lib/ui/hooks/useExportAs'
+export { useIsMultiplayer, useMultiplayerStatus } from './lib/ui/hooks/useIsMultiplayer'
```

Comment:
@SomeHats is it OK to make these public? They are used by the default context menu impl.

Reply:
all good!

Reply:
Should we rename that? Sync status or something? We don't use Multiplayer elsewhere anymore.

Reply:
i don't think we made a decision on this in person so I'll just merge for now and we have until the next sdk release to think of alternatives.

Reply:
PR here: https://github.com/tldraw/tldraw/pull/4349